src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3198,19 +3194,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           Outputs="$(TargetFrameworkMonikerAssemblyAttributesPath)"
           Condition="'$(GenerateTargetFrameworkAttribute)' == 'true'">
 
-    <!-- This is a file shared between projects so we have to take care to handle simultaneous writes (by ContinueOnError)
-             and a race between clean from one project and build from another (by not adding to FilesWritten so it doesn't clean) -->
     <WriteLinesToFile
         File="$(TargetFrameworkMonikerAssemblyAttributesPath)"
         Lines="$(TargetFrameworkMonikerAssemblyAttributeText)"
         Overwrite="true"
-        ContinueOnError="true"
         Condition="'@(Compile)' != '' and '$(TargetFrameworkMonikerAssemblyAttributeText)' != ''"
         />
 
     <ItemGroup Condition="'@(Compile)' != '' and '$(TargetFrameworkMonikerAssemblyAttributeText)' != ''">
       <Compile Include="$(TargetFrameworkMonikerAssemblyAttributesPath)"/>
-      <!-- Do not put in FileWrites: this is a file shared between projects in %temp%, and cleaning it would create a race between projects during rebuild -->
```

Comment:
So it should be in FileWrites now, right? Then you could remove the need for `TargetFrameworkMonikerAssemblyAttributesFileClean`.